src/Nethermind/Nethermind.Monitoring/Metrics/MetricsController.cs

```diff
@@ -117,8 +130,9 @@ private static string BuildGaugeName(string propertyName)
             return Regex.Replace(propertyName, @"(\p{Ll})(\p{Lu})", "$1_$2").ToLowerInvariant();
         }
 
-        private static Gauge CreateGauge(string name, string help = "", GaugeConfiguration configuration = null)
-                => Prometheus.Metrics.CreateGauge($"nethermind_{name}", help, configuration);
+        private static Gauge CreateGauge(string name, string help = "", GaugeConfiguration configuration = null, bool useUserDefinedName = false)
+                => useUserDefinedName ? Prometheus.Metrics.CreateGauge(name, help, configuration) :
+        Prometheus.Metrics.CreateGauge($"nethermind_{name}", help, configuration);
```

Comment:
Move adding `nethermind_` prefix to `BuildGaugeName`, then you don't need `useUserDefinedName` parameter at all.